code/controllers/master.dm

```diff
@@ -120,129 +148,270 @@ var/global/datum/controller/master/Master = new()
 	sleep(1)
 
 	// Loop.
-	Master.process()
+	Master.startprocessing()
 
 // Notify the MC that the round has started.
 /datum/controller/master/proc/RoundStart()
+	round_started = 1
 	var/timer = world.time
-	for(var/datum/subsystem/SS in subsystems)
+	for (var/datum/subsystem/SS in subsystems)
+		if (SS.flags & SS_FIRE_IN_LOBBY || SS.flags & SS_TICKER)
+			continue //already firing
 		timer += world.tick_lag
 		SS.can_fire = 1
 		SS.next_fire = timer + rand(0, SS.wait) // Stagger subsystems.
```

Comment:
Now that you have SS_KEEP_TIMING flag maybe its better to specifically spread those that use it evenly